.gitignore

```diff
@@ -264,4 +264,10 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Prevent config values from being pushed
+qtip.config.json
```

Comment:
It shouldn't be placed in the .gitignore as `qtip.config.json` can be included in heroku and vercel builds, remove this line